src/applications/lgy/coe/status/components/DocumentUploader/FileListItem.jsx

```diff
@@ -11,7 +11,7 @@ const FileListItem = ({ index, file, onClick }) => {
 
   return (
     <div
-      className="vads-u-background-color--gray-lightest vads-u-padding-y--1 vads-u-padding-x--2 vads-u-margin-y--1"
+      className="file-list-item vads-u-background-color--gray-lightest vads-u-padding-y--1 vads-u-padding-x--2 vads-u-margin-y--1"
```

Comment:
Is this supposed to have been added? I'm not seeing it used as a selector or anything

Reply:
Yeah, I was using it in a unit test, but ended up changing it. I'll need to remove this class then.